includes/Order/functions.php

```diff
@@ -49,8 +49,16 @@ function dokan_get_seller_amount_from_order( $order_id, $get_array = false ) {
  *
  * @return array
  */
-function dokan_get_seller_orders( $seller_id, $status = 'all', $order_date = NULL, $limit = 10, $offset = 0, $customer_id = null ) {
+function dokan_get_seller_orders( $request ) {
```

Comment:
This param changes will give an error in https://github.com/weDevsOfficial/dokan/blob/d8cf54a6fef803a5ff97fd0b010ff00cb4c7a15f/includes/Dashboard/Templates/Orders.php#L112

&

https://github.com/weDevsOfficial/dokan/blob/d8cf54a6fef803a5ff97fd0b010ff00cb4c7a15f/includes/Dashboard/Templates/Orders.php#L126